src/screens/crew1/singlePilotScreen.cpp

```diff
@@ -120,6 +121,9 @@ void SinglePilotScreen::onDraw(sf::RenderTarget& window)
 {
     if (my_spaceship)
     {
+        // Toggle ship capabilities.
+        combat_maneuver->setVisible(my_spaceship->getCanCombatManeuver());
```

Comment:
Instead of each station controlling visibility of this control, the control itself could manage its own visibility. Just like the docking button controls it's own labels and so on.

This does require that the guiElement has some kind of update function called even if it is not visible. As it currently does not receive anything if it is hidden. It requires a bit more code changes "up front" but after that is less error prone, and less details in the code.